### PR TITLE
fix for ipad, ported from react-native repo

### LINKED ISSUE #12: Crashes on iPads for iOS 8+
Since the new UIPopoverPresentationController is used for presentation, some additional setup needs to be done.

### PR DESCRIPTION
**Pull request by chirag04**
fixes #12 

cc @isair

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ ActivityView.show({
   text: "Text you want to share",
   url: "URL you want to share",
   imageUrl: "Url of the image you want to share/action",
-  image: "Name of the image in the app bundle"
+  image: "Name of the image in the app bundle",
+  anchor: React.findNodeHandle(this.refs.share), // where to show popup on ipad?
```
Comment (naoufal) [inline README.md]: Minor, but can you update the comment to:

`Where you want the share popup to point to on iPad`

File: README.md [hunk 2]
```diff
@@ -78,14 +80,15 @@ var YourComponent = React.createClass({
 Displays the Activity View with actions relevant to the `shareObject` passed.
 
 __Arguments__
-- `shareObject` - An _Object_ containing one or more of the following keys `text`, `url`, `imageUrl` or `image`.
+- `shareObject` - An _Object_ containing one or more of the following keys `text`, `url`, `anchor`, `imageUrl` or `image`.
```
Comment (naoufal) [inline README.md]: We should specify that anchor is optional.  I'd rather the centered popup be the default behaviour.

**Comment by naoufal** [comment]
@chirag04 aside from that LGTM.

**Comment by chirag04** [comment]
should be ok now i guess.

**Comment by naoufal** [comment]
:rocket: Published to npm.

**Comment by chirag04** [comment]
:beers:

**Pull request merged**